pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -573,6 +580,24 @@ def handle_command_generate_args(  # noqa: C901
             "--fresh",
         ]
         return line
+    elif cmd == "meson":
+        if line[:2] != ["meson", "setup"]:
+            return line
+
+        if "MESON_CROSS_FILE" in os.environ:
+            line[:2] = [
+                "meson",
+                "setup",
+                "--cross-file",
+                os.environ["MESON_CROSS_FILE"],
+            ]
+
+        return line
+    elif cmd in ("install_name_tool", "otool"):
+        # In MacOS, meson tries to run install_name_tool to fix the rpath of the shared library
+        # assuming that it is a ELF file. We need to skip this step.
+        # See: https://github.com/mesonbuild/meson/issues/8027
+        return ["echo", *line]
```

Comment:
I think if we don't want to run the code we can just return `None` here. Is there a reason you want to echo it?

Reply:
It was to make those commands be shown in the build.log, which makes debugging easier.